google/cloud/spanner_dbapi/batch_executor.py

```diff
@@ -61,16 +112,20 @@ def execute_statement(self, parsed_statement: ParsedStatement):
             raise ProgrammingError("Only DML statements are allowed in batch DML mode.")
         self._statements.append(parsed_statement.statement)
 
-    def run_batch_dml(self):
+    def run_batch(self):
         """Executes all the buffered statements on the active dml batch by
         making a call to Spanner.
+
+        This method is internal and not for public use as it can change anytime.
```

Comment:
nit: here and elsewhere

```suggestion
        This method is internal and not for public use
```

Reply:
Done

Reply:
Could it be that you have not pushed that change yet?

Reply:
Yes I did not. Thanks done